infra/examples/gcp-hris/main.tf

```diff
@@ -31,14 +31,15 @@ resource "google_project" "psoxy-project" {
 
 
 module "psoxy-gcp-bulk" {
-  source = "../../modules/gcp-bulk"
+  source = "git::https://github.com/worklytics/psoxy//infra/modules/gcp-bulk?ref=v0.3.0-beta.3"
 
   project_id           = google_project.psoxy-project.project_id
   worklytics_sa_emails = var.worklytics_sa_emails
   region               = var.region
   bucket_prefix        = var.bucket_prefix
   bucket_location      = var.bucket_location
   source_kind          = var.source_kind
+  psoxy_basedir        = var.psoxy_basedir
```

Comment:
i think better to be consistent with naming of the `aws` case, which I think is `path_to_jar` or something. 

imho, this `gcp-bulk` (and `aws-bulk`) need refactoring so that they don't invoke mvn build again.

Reply:
yes, I also think the aws-hris, aws-bulk need more changes, but not critical right now.